lux/utils/debug_utils.py

```diff
@@ -31,6 +32,29 @@ def show_versions(return_string: bool = False) -> Optional[str]:
     header = "Package Versions\n----------------\n"
     jupyter_versions_str = subprocess.check_output(["jupyter", "--version"])
     jupyter_versions = re.findall(r"(\S+)\s+: (.+)\S*", jupyter_versions_str.decode("utf-8"))
+
+    str_lux_error = ""
+    str_lux_error += (
+        "'lux' library is not installed. You may need to run the following code in your command line:\n"
+    )
+    str_lux_error += "  pip install lux-api"
+
+    # Check if correct lux library is installed
+    try:
+        import lux
+    except ModuleNotFoundError:
+        print(str_lux_error)
+
+    str_upgrade = ""
+    str_upgrade += "The upgraded version of lux is not installed."
+    str_upgrade += "You may need to run the following code in your command line:\n"
+    str_upgrade += "  pip install --upgrade lux-api"
+    lux_version = lux.__version__
+
+    # Check if lux needs to be upgraded
+    if version.parse(lux_version) < version.parse("0.3"):
```

Comment:
We can express this directly as str(lux.__version__) to avoid the use of the `packaging` library.